collections/business/general-notifications/event-general-notifications/account-statement-ready/account-statement-ready/in-app-notification/in-app_channel-settings.json

```diff
@@ -1,5 +1,5 @@
 {
+  "$schema": "../../../../../../../schema/v1/in-app-channel-settings.json",
   "level": "SUCCESS",
-  "targetGroup": "USER",
   "origin": "AccountStatement"
```

Comment:
@ajitavbb @eacarvalho can we remove this and get it from `bbLocation` header or is that not reliable?